logdna/logdna.py

```diff
@@ -11,6 +11,9 @@
 from .utils import sanitize_meta, get_ip
 
 logger = logging.getLogger(__name__)
+internal_handler = logging.FileHandler('LogDNAHandler.log')
```

Comment:
No directory specification? What if customer installs it with `sudo pip install logdna` and package goes into `/...` place and customer starts using it without `sudo`, isn't this gonna cause `Permission Denied` issue? Can you double-check this?

Reply:
Good point. I will specify a directory.

Reply:
Actually, this function creates a file in the existing directory only. It can't create a new directory. And we can't know in advance which directories a user will have on his machine